pkgs/by-name/op/opensoldat/package.nix

```
@@ -13,30 +13,40 @@
   xorg,
   autoPatchelfHook,
   cmake,
+  python3,
 }:
 
 let
   base = stdenv.mkDerivation rec {
     pname = "opensoldat-base";
-    version = "unstable-2021-09-05";
+    version = "unstable-2022-10-29";
 
     src = fetchFromGitHub {
       name = "base";
       owner = "opensoldat";
       repo = "base";
-      rev = "6c74d768d511663e026e015dde788006c74406b5";
-      sha256 = "175gmkdccy8rnkd95h2zqldqfydyji1hfby8b1qbnl8wz4dh08mz";
+      rev = "5f880cdee5b50168d9d4d79026623b5624af9e40";
+      sha256 = "sha256-nRUg4L6gfzz9dCxGs4pr3Xr0DVcuKn6At82ALOgG8IQ=";
     };
 
-    nativeBuildInputs = [ zip ];
+    nativeBuildInputs = [ python3 ];
 
     buildPhase = ''
-      sh create_smod.sh
+      python create_smod.py
     '';
 
+    patches = [
+      (fetchpatch {
+        # https://github.com/opensoldat/base/pull/27
+        name = "zip-timestamps.patch";
+        url = "https://github.com/opensoldat/base/pull/27/commits/5d5312728e8a5a455b1f0be8b75ffe096573dfdb.patch";
```

We should use this URL which less likely to disappear.

```suggestion
        url = "https://github.com/opensoldat/base/commit/5d5312728e8a5a455b1f0be8b75ffe096573dfdb.patch";
```